ghost/i18n/test/i18n.lint.js

```diff
@@ -418,28 +425,23 @@ function analyzeSingleTranslation(key, translated, context) {
 
     const {errors, variables: used} = parseTranslationString(translated);
 
-    if (reportInvalidTranslations) {
+    {
```

Comment:
This block needs to be retained to avoid redeclaration of const variables All of the other cases are nested under an if/for bock